exercises/robot-simulator/defs.go

```diff
@@ -19,14 +19,14 @@ type Command byte // valid values are 'R', 'L', 'A'
 type RU int
 type Pos struct{ Easting, Northing RU }
 type Rect struct{ Min, Max Pos }
-type DirAt struct {
+type Robot struct {
 	Dir
 	Pos
 }
 
 // additional definition used in step 3
 
-type Place struct {
+type Robot3 struct {
```

Comment:
The numbers after each robot was supposed to represent each step of the problem. So in Step 3 you would use the Robot3 and in Step 1 you would need Robot1, making it similar to how Room3 and Robot3 were original used. 

NamedRobot is a pretty good name. If we assume that a robot is defined by a position and direction than NamedRobot would imply location as well.